omnigibson/robots/franka.py

```diff
@@ -84,9 +86,93 @@ def __init__(
+            self._ag_start_points = [
+                GraspingPoint(link_name=f"base_link", position=[0.015, 0, -0.03]),
+                GraspingPoint(link_name=f"base_link", position=[0.015, 0, -0.08]),
+                GraspingPoint(link_name=f"link_15_0_tip", position=[0, 0.015, 0.007]),
+            ]
+            self._ag_end_points = [
+                GraspingPoint(link_name=f"link_3_0_tip", position=[0.012, 0, 0.007]),
+                GraspingPoint(link_name=f"link_7_0_tip", position=[0.012, 0, 0.007]),
+                GraspingPoint(link_name=f"link_11_0_tip", position=[0.012, 0, 0.007]),
+            ]
+        elif "leap" in end_effector:
+            self._model_name = f"franka_{end_effector}"
+            self._eef_link_names = "palm_center"
+            # thumb.proximal, ..., thumb.tip, ..., ring.tip
+            self._finger_link_names = [
+                f"{link}_{i}" for i in range(1, 5) for link in ["mcp_joint", "pip", "dip", "fingertip", "realtip"]
+            ]
+            self._finger_joint_names = [
+                f"finger_joint_{i}" for i in [12, 13, 14, 15, 1, 0, 2, 3, 5, 4, 6, 7, 9, 8, 10, 11]
+            ]
+            # position where the hand is parallel to the ground
+            self.__default_joint_pos = np.concatenate(([0.86, -0.27, -0.68, -1.52, -0.18, 1.29, 1.72], np.zeros(16)))
+            self._teleop_rotation_offset = np.array([-0.7071, 0.7071, 0, 0])
+            self._ag_start_points = [
+                GraspingPoint(link_name=f"palm_center", position=[0, -0.025, 0.035]),
+                GraspingPoint(link_name=f"palm_center", position=[0, 0.03, 0.035]),
+                GraspingPoint(link_name=f"fingertip_4", position=[-0.0115, -0.07, -0.015]),
+            ]
+            self._ag_end_points = [
+                GraspingPoint(link_name=f"fingertip_1", position=[-0.0115, -0.06, 0.015]),
+                GraspingPoint(link_name=f"fingertip_2", position=[-0.0115, -0.06, 0.015]),
+                GraspingPoint(link_name=f"fingertip_3", position=[-0.0115, -0.06, 0.015]),
+            ]
+        elif end_effector == "inspire":
+            self._model_name = f"franka_{end_effector}"
+            self._eef_link_names = "palm_center"
+            # thumb.proximal, ..., thumb.tip, ..., ring.tip
+            hand_part_names = [11, 12, 13, 14, 21, 22, 31, 32, 41, 42, 51, 52]
+            self._finger_link_names = [f"link{i}" for i in hand_part_names]
+            self._finger_joint_names = [f"joint{i}" for i in hand_part_names]
+            # position where the hand is parallel to the ground
+            self.__default_joint_pos = np.concatenate(([0.86, -0.27, -0.68, -1.52, -0.18, 1.29, 1.72], np.zeros(12)))
+            self._teleop_rotation_offset = np.array([0, 0, 0.707, 0.707])
+            # TODO: add ag support for inspire hand
+            self._ag_start_points = [
+                # GraspingPoint(link_name=f"base_link", position=[0, -0.025, 0.035]),
```

Comment:
are the grasping points not tuned yet?

Reply:
yes, there seems to be a bug related to ag, will update this with another PR when ag is fixed